### DISCUSSION TITLE
brew list --pinned

### DISCUSSION THREAD

**Discussion by Bruno21** [Homebrew/discussions#112] [Everyday usage] [answered]
Hello,

```
Updated Homebrew from 5465a866a to dbdf7cf26.
...
```

Since today, I got this error:

```
❯ brew list --pinned
Usage: brew list, ls [options] [formula|cask]
...
Error: Invalid usage: `pinned` cannot be passed without `formula`.

```

**Comment by SMillerDev** [comment]
I'm not sure what the question is.

**Reply by Bruno21** [reply]
Sorry. 
When I run `brew list --pinned`, i expect the 'pinned' packages list.  I have this error instead (since today)

**Reply by SMillerDev** [reply]
Because the parameters got changed. You would now need to pass `brew list --formulae --pinned` I believe

**Comment by Bruno21** [comment, marked as answer]
Perfect !
Thanks.
